mlflow/langchain/__init__.py

```diff
@@ -85,6 +87,15 @@
 FLAVOR_NAME = "langchain"
 _MODEL_TYPE_KEY = "model_type"
 
+# The following constants is used in flavor configuration to specify the code paths
+# that should be set using _set_config_path when the model is loaded by code.
+_CODE_CONFIG = "_code_config"
+__code_paths__ = None
+
+
+def _set_config_path(path):
+    globals()["__code_paths__"] = path
```

Comment:
Shouldn't this be called `config_path`?

```suggestion
    globals()["__databricks_rag_config_path__"] = path
```